Cannot find module 'flat-ui' from...

`var flat-ui = require('flat-ui');`
this is the code in my js file.
and i have alreadly
`npm install flat-ui --save-dev`
and i want to use the browserify.
but when i execute "gulp browserify"
this is the error
```
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Cannot find module 'flat-ui' from 'd:\home\workspace\jngongzuo.com\src\js
'
    at d:\node_modules\browserify\node_modules\resolve\lib\async.js:46:17
    at process (d:\node_modules\browserify\node_modules\resolve\lib\async.js:173
:43)
    at ondir (d:\node_modules\browserify\node_modules\resolve\lib\async.js:188:1
7)
    at load (d:\node_modules\browserify\node_modules\resolve\lib\async.js:69:43)

    at onex (d:\node_modules\browserify\node_modules\resolve\lib\async.js:92:31)

    at d:\node_modules\browserify\node_modules\resolve\lib\async.js:22:47
    at FSReqWrap.oncomplete (fs.js:95:15)
```

--- a/index.js
+++ b/index.js
@@ -1,0 +1,2 @@
+// You can require() this file in a CommonJS environment.
+require('./dist/js/flat-ui.js');

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "author": "Designmodo, Inc.",
   "license": "CC BY 3.0 and MIT",
   "homepage": "http://designmodo.github.io/Flat-UI/",
+  "main": "index",
   "keywords": [
     "ui",
     "flat",

